.ci/container_setup.d/10-install-signing-service.sh

```diff
@@ -2,14 +2,32 @@
 
 set -eu
 
-if pulp --format none --refresh-api --base-url "http://localhost:8080" ${PULP_API_ROOT:+--api-root "${PULP_API_ROOT}"} --username "admin" --password "password" debug has-plugin --name "deb"
+has_deb=$(pulp --refresh-api --base-url "http://localhost:8080" ${PULP_API_ROOT:+--api-root "${PULP_API_ROOT}"} --username "admin" --password "password" debug has-plugin --name "deb")
+has_ansible=$(pulp --base-url "http://localhost:8080" ${PULP_API_ROOT:+--api-root "${PULP_API_ROOT}"} --username "admin" --password "password" debug has-plugin --name "ansible")
+if $has_deb || $has_ansible
```

Comment:
Now you are executing the output of these commands. I see why it works, but my instinct whines "this is unsafe".
What if you do
`pulp ... has-plugin ... && HAS_DEB=true || HAS_DEB=""`
and then check with
`if [ "$HAS_DEB" ]; then`
?

Reply:
I tested these commands with a custom script, and it properly knows the difference between true and false. I can change it if you really want as I'm not an expert in bash scripting.

Reply:
Yes, i know, why it works. `true` and `false` are standard unix commands. And we call the content of the variable here which just so happens to be either of them. This is on the same level as `eval` in python (for some reason very close to "evil"). And to me that is a anti pattern. So yes, i'd like it to be changed.
`[` on the other hand is an alias for the bash builtin `test` so we only pass the content of the variable as a parameter.